packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -170,26 +170,29 @@ export function PrivateBlockToolbar( {
 				{ isUsingBindings && canBindBlock( blockName ) && (
 					<BlockBindingsIndicator />
 				) }
-				{ ( shouldShowVisualToolbar || isMultiToolbar ) &&
-					isDefaultEditingMode && (
```

Comment:
Removing this condition makes the block toolbar controls visible regardless of the editing mode. It affects part of the Product Editor UI. [Here is the issue](https://github.com/woocommerce/woocommerce/issues/45908).

Before this change, when the editing mode was contentOnly, the controls that didn't allow editing the block content were not visible.
Below, the custom TextArea field block shows the alignment and regular format buttons.

<img width="691" alt="image" src="https://github.com/WordPress/gutenberg/assets/77539/d459ce0c-975d-4b74-b6d0-15d79d509da5">

After this change, the toolbar shows the parent block, which doesn't do anything since its template is locked to contentOnly:

<img width="481" alt="Screenshot 2024-04-01 at 16 43 25" src="https://github.com/WordPress/gutenberg/assets/77539/c192e716-26fc-4b5e-9229-080d04c36ae0">

Could we consider it a breaking change in terms of the UI?
How could we hide it, recreating the previous state before merging these changes?

Reply:
Thanks for flagging. I'll open a fix.

Reply:
I've opened https://github.com/WordPress/gutenberg/pull/60647 which dials this change back to only show the icon on pattern blocks with `contentOnly`.

Reply:
Nice! Thanks a bunch. I see it's already 🍏